examples/cadvisor-influx/juttle_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "adapters": {
+        "juttle-influxdb-adapter": {
```

Comment:
Oh actually, you don't need this file in this directory at all. We only have the unified config in the examples/ directory.